calabash/Classes/FranklyServer/Routes/LPConditionRoute.m

```diff
@@ -199,7 +204,9 @@ - (BOOL) atLeastOneAnimatingOnMainThreadWithQuery:(id) query {
                                                       NSUInteger idx,
                                                       BOOL *stop) {
             CAAnimation *animation = [view.layer animationForKey:key];
-            if (animation.duration > kLPConditionRouteAnimationDurationLimit) {
+            if ([key  isEqual: @"UITextSelectionViewCaretBlinkAnimation"]) {
+              // do nothing
```

Comment:
It feels weird having an empty if-case. An alternative approach could be to extract the boolean condition into it's own method and just have an if. So you would have

```
- (BOOL) keyIsNotBlinkingAndAnimationDurationIsHigherThanConditionRouteAnimationDurationLimit {
    return ![key isEqual: @"..."] && animation.duration > klPCondi....
}
```
Is this better? I don't know. Having ridiculously long method names is the Apple way.